web/locales/de.json

```diff
@@ -51,11 +52,13 @@
     "legends": {
         "windpotential": "Potenzial Windenergie",
         "solarpotential": "Potenzial Solarenergie",
-        "colorblindmode": "farbenblind Modus"
+        "colorblindmode": "farbenblind Modus",
```

Comment:
I'd say `Modus für Farbenblinde`. It could be as well `Farbenblindmodus`, but I prefer the former.